RUNIT TEST PROTOCOL -- Sat Sep 11 08:26:25 2010 
*********************************************** 
Number of test functions: 3 
Number of errors: 1 
Number of failures: 0 

 
1 Test Suite : 
RProtoBuf unit testing - 3 test functions, 1 error, 0 failures
ERROR in /tmp/Rinst2001904304/RProtoBuf/unitTests/runit.addressbook.R: Error while sourcing  /tmp/Rinst2001904304/RProtoBuf/unitTests/runit.addressbook.R : Error in parse(n = -1, file = file) : 30:0: unexpected end of input
28:     checkEquals(book$person[[2]]$phone[[1]]$type,   0,                   msg="Second person phone number")
29: }
   ^



Details 
*************************** 
Test Suite: RProtoBuf unit testing 
Test function regexp: ^test.+ 
Test file regexp: ^runit.+\.[rR]$ 
Involved directory: 
/tmp/Rinst2001904304/RProtoBuf/unitTests 
--------------------------- 
Test file: /tmp/Rinst2001904304/RProtoBuf/unitTests/runit.addressbook.R 
/tmp/Rinst2001904304/RProtoBuf/unitTests/runit.addressbook.R: ERROR !! 
Error while sourcing  /tmp/Rinst2001904304/RProtoBuf/unitTests/runit.addressbook.R : Error in parse(n = -1, file = file) : 30:0: unexpected end of input
28:     checkEquals(book$person[[2]]$phone[[1]]$type,   0,                   msg="Second person phone number")
29: }
   ^
--------------------------- 
Test file: /tmp/Rinst2001904304/RProtoBuf/unitTests/runit.golden.message.R 
test.import: (5 checks) ... OK (0 seconds)
test.readProtoFile: (61 checks) ... OK (0.01 seconds)
